magento/inventory
# MSI-2138: Disable fields in Mass action -> Update attributes -> Advanced inventory.

### Description (*)


### Fixed Issues (if relevant)

1. https://github.com/magento/inventory/issues/2138: Disable fields in Mass action -> Update attributes -> Advanced inventory

### Manual testing scenarios (*)

1. see https://github.com/magento/inventory/issues/2138
2. Create simple product in single source mode.
3. Try mass update quantity and stock status.
4. Verify, product correctly appears and disappears on storefront.
5. Add source and stock.
6. Create another simple product with only additional source.
7. Mass update 'Manage Stock'.
8. Verify default source isn't connected to this simple product after update.

### Notice:    
All operations should be performed with ./bin/magento cron:run as inventory update is asynchronous.

### Questions or comments


### Contribution checklist (*)
